util/start_test

```diff
@@ -1092,14 +1092,15 @@ def set_up_executables():
 
     # pre-diff
     if args.prediff:
-        if os.path.isfile(args.prediff) and os.access(args.prediff, os.X_OK):
-            args.prediff = os.path.abspath(args.prediff)
-            logger.write("[system-wide prediff: {0}]".format(args.prediff))
-        else:
-            logger.write("[Error: Cannot find or execute system-wide prediff: "
-                    "{0}".format(args.prediff))
-            finish()
-        os.environ["CHPL_SYSTEM_PREDIFF"] = args.prediff
+        for pdf in args.prediff:
```

Comment:
Nit, but I find `pdf` to be a confusing short name for prediff. I'd probably just keep it as `prediff`